sjsonnet/src-js/sjsonnet/Platform.scala

```diff
@@ -43,7 +43,7 @@ object Platform {
   }
 
   def yamlToJson(s: String): ujson.Value = {
-    val docs = s.split("---\\s*\n")
+    val docs = s.split("(^|\n)---\\s*(\n|$)")
```

Comment:
We can compile the pattern and reuse it.

Reply:
Fair enough. Done. PTAL